akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/MultipartUnmarshallersSpec.scala

```diff
@@ -261,21 +279,24 @@ class MultipartUnmarshallersSpec extends FreeSpec with Matchers with BeforeAndAf
               RawHeader("Content-Additional-1", "anything"),
               RawHeader("Content-Additional-2", "really-anything")))) // verifies order of headers is preserved
       }
-      // TODO: reactivate after multipart/form-data unmarshalling integrity verification is implemented
-      //
-      //      "reject illegal multipart content" in {
-      //        val Left(MalformedContent(msg, _)) = HttpEntity(`multipart/form-data` withBoundary "XYZABC", "--noboundary--").as[MultipartFormData]
-      //        msg shouldEqual "Missing start boundary"
-      //      }
-      //      "reject illegal form-data content" in {
-      //        val Left(MalformedContent(msg, _)) = HttpEntity(`multipart/form-data` withBoundary "XYZABC",
-      //          """|--XYZABC
-      //            |content-disposition: form-data; named="email"
-      //            |
-      //            |test@there.com
-      //            |--XYZABC--""".stripMargin).as[MultipartFormData]
-      //        msg shouldEqual "Illegal multipart/form-data content: unnamed body part (no Content-Disposition header or no 'name' parameter)"
-      //      }
+      "multipartFormDataUnmarshaller should reject" - {
+        "illegal multipart content" in {
+          Await.result(
+            Unmarshal(HttpEntity(`multipart/form-data` withBoundary "XYZABC", "--noboundary--"))
+              .to[Multipart.FormData].failed, 1.second).getMessage shouldEqual "Unexpected end of multipart entity"
```

Comment:
Yeah. Just came across these two and reenabled them as well. Looks like the current stage base multipart parser doesn't provide the "Missing start boundary" message anymore. Same for "unnamed body part (no Content-Disposition header or no 'name' parameter)" -> "multipart/form-data part must contain `Content-Disposition` header with `name` parameter". (/cc @sirthias)